src/ion/data_structures.rs

```diff
@@ -440,7 +440,7 @@ impl<'a, F: Function> Env<'a, F> {
 #[derive(Clone, Debug)]
 pub struct SpillSlotData {
     pub ranges: LiveRangeSet,
-    pub class: RegClass,
+    pub size: u32,
```

Comment:
can we call this field `slots` to make the units clear (as opposed to e.g. bytes)?